cmd/juju/commands/bootstrap.go

```diff
@@ -358,7 +358,9 @@ func (c *bootstrapCommand) Init(args []string) (err error) {
 		return errors.New("--db-snap-asserts requires --db-snap")
 	}
 
-	if c.ControllerCharmPath != "" {
+	if strings.HasPrefix(c.ControllerCharmPath, "ch:") {
```

Comment:
What we do elsewhere for deploy is not require "ch:". We check to see if the param is a filename (absoluete path or starts with "./blah") and if not interpret as a ch charm. We should be consistent

Reply:
Ok fair enough. Maybe we could also rename the flag to `--controller-charm`